localization/ndt_scan_matcher/src/map_update_module.cpp

```diff
@@ -107,6 +107,20 @@ bool MapUpdateModule::should_update_map(
   return distance > param_.update_distance;
 }
 
+bool MapUpdateModule::out_of_map_range(const geometry_msgs::msg::Point & position)
```

Comment:
`out_of_map` and `should_update_map` have overlapping content. Can you make them common?

Reply:
I feel the same way, but there are some minor differences so I am not sure whether the function can be used as common. I am still thinking about it.

Reply:
I cannot come up with any good ideas to organize the code. :disappointed: 
I think it is okay to merge this as is.

Reply:
Since there is little duplication of code, I also think it is acceptable as is in this time. :ok_man: